mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/avtale/db/AvtaleQueriesTest.kt

```diff
@@ -729,15 +738,16 @@ class AvtaleQueriesTest : FunSpec({
                 )
                 queries.upsert(avtaleAktiv)
 
-                val avtaleAvsluttet = AvtaleFixtures.oppfolging.copy(
+                val avtaleAvsluttet = AvtaleFixtures.AFT.copy(
                     id = UUID.randomUUID(),
                     detaljerDbo = AvtaleFixtures.detaljerDbo().copy(
                         status = AvtaleStatusType.AVSLUTTET,
                     ),
+                    prismodellDbo = AvtaleFixtures.prismodellDbo(),
```

Comment:
Ser ut som at det er veldig mange steder i testene her der `prismodellDbo = AvtaleFixtures.prismodellDbo(),` settes, men dette er vel allerede en default value i selve `AvtaleFixtures.AFT` (i dette eksempelet)?

Supert om vi ikke setter flere felter i disse modellene hvis ikke trengs for selve test caset (så er det lettere å tolka hva som er "under test")! :smile: